workflow/run_analysis.rb

```diff
@@ -477,11 +485,49 @@ def run_workflow(yml, in_threads, measures_only, debug_arg, overwrite, building_
   return true
 end
 
-def create_lib_folder(lib_dir, resources_dir, housing_characteristics_dir)
+def checksum_dir_content(directory_path)
+  files = Dir.glob('**/*', base: directory_path).select { |fn| File.file?(File.join(directory_path, fn)) }
```

Comment:
@jmarrec I had to prepend `directory_path` here so that the size of the files array isn't zero. Not sure how you're testing/application worked without this fixed?